test/conformance/api/v1/errorcondition_test.go

```diff
@@ -174,20 +174,19 @@ func TestContainerExitingMsg(t *testing.T) {
 
 			t.Log("When the containers keep crashing, the Configuration should have error status.")
 
-			err := v1test.WaitForConfigurationState(clients.ServingClient, names.Config, func(r *v1.Configuration) (bool, error) {
+			if err := v1test.WaitForConfigurationState(clients.ServingClient, names.Config, func(r *v1.Configuration) (bool, error) {
 				cond := r.Status.GetCondition(v1.ConfigurationConditionReady)
-				if cond != nil && !cond.IsUnknown() {
+				if cond != nil && cond.IsFalse() {
```

Comment:
I don't understand this change.  This means if the Revision becomes ready we will keep waiting...

Reply:
We should not really become ready here, do we?